src/pages/u/[handle].tsx

```diff
@@ -0,0 +1,37 @@
+import { UserAPI } from '@/lib/api';
+import withAccessType from '@/lib/hoc/withAccessType';
+import { CookieService, PermissionService } from '@/lib/services';
+import type { PublicProfile } from '@/lib/types/apiResponses';
+import { CookieType } from '@/lib/types/enums';
+import type { GetServerSideProps } from 'next/types';
+
+interface UserProfilePageProps {
+  user: PublicProfile;
+}
+
+const UserProfilePage = ({ user }: UserProfilePageProps) => {
+  return <pre>{JSON.stringify(user, null, 2)}</pre>;
+};
+
+export default UserProfilePage;
+
+const getServerSidePropsFunc: GetServerSideProps = async ({ params, req, res }) => {
+  const handle = params?.handle as string;
+  const token = CookieService.getServerCookie(CookieType.ACCESS_TOKEN, { req, res });
+
+  try {
+    const user = await UserAPI.getUserByHandle(token, handle);
+    return {
+      props: {
+        user,
+      },
+    };
+  } catch (err: any) {
+    return { redirect: { destination: '/', permanent: false } };
```

Comment:
nit: avoid magic values in our codebase to make it clear what the intended page is supposed to be through the variable naming and make the codebase much more maintainable if we ever need to shift the page route structure 
use `config.homeRoute` instead of `'/'`